app/javascript/components/aggregate_status_card.jsx

```diff
@@ -1,7 +1,7 @@
 import React, { useEffect, useState } from 'react';
 import PropTypes from 'prop-types';
 
-import { Quadicon } from '@manageiq/react-ui-components/dist/quadicon';
+import { Quadicon } from './quadicon';
```

Comment:
quadicons are no longer being used in the product, i feel that we should review/cleanup any leftover quadicon related code from ui repo. @Fryguy what do you think, i don't see nay point in moving over the component that is not even in use.